### PR TITLE
Banner

### LINKED ISSUE #214: Warning bar in acceptance environment
It happens sometimes that clients are using the acceptance environment when thinking that they are at production/live.  To prevent mistakes it would be good to see a clear warning in Ginger (YOU ARE WORKING IN ACCEPTANCE). Is it possible to have a standard warning bar (like a cookie bar) in all sites that reminds you of this. (maybe even possible at the frontend of the site too).

### PR DESCRIPTION
**Pull request by DorienD**
Fix #214.

- [x] Add unpublished banner rsc in schema
- [x] Change template check to look for (published!) banner rsc as well
- [ ] Add a close button
- [ ] Add a cookie on the close button

### CODE REVIEW

**Comment by DorienD** [comment]
@ddeboer i need some help with the prio of this module. I have it on 500 now but it stills adds the _html_head.tpl after the base _html_head.tpl. Which is giving conflicts with my styles.

File: modules/mod_ginger_banner/mod_ginger_banner.erl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-module(mod_ginger_banner).
+-author("Driebit <info@driebit.nl>").
+
+-mod_title("Ginger banner module").
+-mod_description("Shows a banner on top of the site, if this is added in the config of the site or trough the rsc banner.").
+-mod_prio(500).
```
Comment (ddeboer) [inline modules/mod_ginger_banner/mod_ginger_banner.erl]: Should be a *lower* number than mod_ginger_base (which is 250): see also the [docs](http://docs.zotonic.com/en/latest/developer-guide/modules.html#priority).

**Comment by ddeboer** [comment]
Yep, would be nice, but is not strictly necessary. Let’s start with this simple version and see how editors use it before we start adding more features.

**Comment by DorienD** [comment]
@ddeboer can you merge this, if you think this is good.

**Comment by ddeboer** [comment]
Missing a readme: see [guidelines](https://github.com/driebit/ginger/blob/master/docs/guidelines.md#14-create-useful-ginger-modules).

**Comment by DorienD** [comment]
Readme added! Now the only thing that remains is setting a cookie and a close button so you can click it away.

**Pull request merged**